### PR TITLE
Added option to hide the banner

### PR DESCRIPTION
**Pull request by samirettali**

### CODE REVIEW

File: help.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func Usage() {
 		Description:   "",
 		Flags:         make([]UsageFlag, 0),
 		Hidden:        false,
-		ExpectedFlags: []string{"ac", "acc", "c", "maxtime", "maxtime-job", "p", "s", "sa", "se", "sf", "t", "v", "V"},
+		ExpectedFlags: []string{"ac", "acc", "c", "maxtime", "maxtime-job", "nb", "p", "s", "sa", "se", "sf", "t", "v", "V"},
```
Comment (joohoi) [inline help.go:64]: In case we go with writing to `stderr`, we don't need to add a new CLI flag.

File: main.go [hunk 2]
```diff
@@ -104,6 +104,7 @@ func main() {
 	flag.StringVar(&opts.outputFormat, "of", "json", "Output file format. Available formats: json, ejson, html, md, csv, ecsv")
 	flag.StringVar(&conf.OutputDirectory, "od", "", "Directory path to store matched results to.")
 	flag.BoolVar(&conf.IgnoreBody, "ignore-body", false, "Do not fetch the response content.")
+	flag.BoolVar(&conf.NoBanner, "nb", false, "Do not print the banner ")
```
Comment (joohoi) [inline main.go:107]: In case we go with writing to `stderr`, we don't need to add a new CLI flag.

File: pkg/ffuf/config.go [hunk 3]
```diff
@@ -21,6 +21,7 @@ type Config struct {
 	OutputFile             string                    `json:"outputfile"`
 	OutputFormat           string                    `json:"outputformat"`
 	IgnoreBody             bool                      `json:"ignorebody"`
+	NoBanner               bool                      `json:"nobanner"`
```
Comment (joohoi) [inline pkg/ffuf/config.go:24]: In case we go with writing to `stderr`, we don't need to add a new configuration variable.

**Comment by samirettali** [comment]
Hi, sorry for the delay, I've been quite busy lately.
The reason I added the option is because when writing writeups from CTFs I almost always remove the banner because it's too big and it makes the great majority of the output.

**Comment by joohoi** [comment]
Ok, that sounds reasonable. I'd still like to output the banner to Stderr, and I think it'd be a good match to make that change in context of this PR. Could you additionally also change the usage text on `README.md` to match the changes. Oh and add yourself to the alphabetically correct place in `CONTRIBUTORS.md` :)

**Pull request closed without merging**